src/python/CRABInterface/Utilities.py

```diff
@@ -121,40 +121,45 @@ def retrieveConfig(externalLink):
         
         return jsonConfig
 
-    extConfCommon = json.loads(retrieveConfig(extconfigurl))
-
-    # below 'if' condition is only added for the transition period from the old config file to the new one. It should be removed after some time.
-    if 'modes' in extConfCommon:
```

Comment:
Naive question: Is it ok to remove this `if`? are we guaranteed that this key will always be in the dictionary?

Reply:
well..we write that dictionary in https://gitlab.cern.ch/crab3/CRAB3ServerConfig/-/blob/master/cmsweb-rest-config.json
so we better follow our own instructions. We simply do not need to have code compatible with two different formats of that file